### ISSUE TITLE
Delta Time color is not working.

### ISSUE THREAD

**Issue by JasonHardman** [NickThissen/iRacingAdminSync#5]
The Delta Time color is always black which is quite hard to read.

**Comment by NickThissen** [comment]
Thanks Jason. I added that delta time color just before I uploaded the source code and never tested it. Does it work correctly now? The idea was to have it white per default but gradually turn red as drivers are closer together (within some limit configurable in the settings).

**Comment by JasonHardman** [comment]
It works ok but it does need a tweak.  The color goes white -> dark red -> bright red.  It would be better if it was light red instead of dark red.  The dark red looks like a combination of red and black, when it would be better if red and white since white is the default.

**Comment by NickThissen** [comment]
Yeah that makes sense, I never thought about that, it should fade from
white to red not black to red. Thanks.

2015-02-25 21:07 GMT+01:00 Jason Hardman notifications@github.com:

> It works ok but it does need a tweak. The color goes white -> dark red ->
> bright red. It would be better if it was light red instead of dark red. The
> dark red looks like a combination of red and black, when it would be better
> if red and white since white is the default.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/NickThissen/iRacingAdminSync/issues/5#issuecomment-76044853
> .

## 

Nick

**Comment by JasonHardman** [comment]
You said it better :)
